plugins/cluster-status-backend/src/statusCheck/kubernetesApi.ts

```diff
@@ -0,0 +1,41 @@
+import { Config } from "@backstage/config";
+import { CustomObjectsApi, KubeConfig } from "@kubernetes/client-node"
+import { Logger } from 'winston';
+
+export const getCustomObjectsApi = (clusterConfig: Config, logger: Logger): CustomObjectsApi => {
+  const clusterToken = clusterConfig.getOptionalString('serviceAccountToken');
+  const kubeConfig = new KubeConfig();
+
+  if (!clusterToken) {
+    logger.info('Using default kubernetes config');
+    kubeConfig.loadFromDefault();
+    return kubeConfig.makeApiClient(CustomObjectsApi);
+  }
+
+  logger.info('Loading kubernetes config from config file');
+  const cluster = {
+    name: clusterConfig.getString('name'),
+    server: clusterConfig.getString('url'),
+    skipTLSVerify: clusterConfig.getOptionalBoolean('skipTLSVerify') ?? false,
+    caData: clusterConfig.getOptionalString('caData'),
+  };
+
+  const user = {
+    name: 'backstage',
```

Comment:
Question: Where this value comes from? Is it arbitrary value? Any value would be accepted?

Reply:
Yea I think its an arbitrary value, for the [Kubernetes context](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#define-clusters-users-and-contexts).